R/variable-to-R.R

```diff
@@ -53,6 +53,11 @@ parse_column <- list(
     datetime=function (col, variable, mode) {
         out <- columnParser("text")(col)
         return(from8601(out))
+    },
+    boolean=function (col, variable, mode) {
```

Comment:
I've made the test on line 172 of test-expressions a bit more explicit (reference the dataframe that created the dataset, and ensure those are equal instead of just grabbing the vector). Is that what you're looking for here, or something else?